src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -275,12 +273,8 @@ export class CalciteAlert {
   private determineActiveAlert(): void {
     if (this.queue?.[0] === this.el) {
       this.openAlert();
-      clearTimeout(this.autoDismissTimeout);
```

Comment:
I think there should be some cases where we still need to clear the timeout.

- When the component is destroyed
- If the autoDismissDuration is changed?